mixins/_icons.scss

```diff
@@ -69,16 +73,19 @@
         content: "\64";
     }
     @if $symbol == 'down' {
-        content: "\76";
+        content: "\56";
```

Comment:
any reason for this change? is this being used in any IAs that might be affected?